mitiq/interface/mitiq_qiskit/tests/test_conversions_qiskit.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 import pytest
 import qiskit
+import qiskit.quantum_info
```

Comment:
Is this needed?

Reply:
I expected `make-format` would have cleaned this, but it's still there for some reasons.
p/s: i removed the unused import.

Reply:
I wonder if `ruff` can't know that since it's possible that `qiskit.quantum_info` could be an optional dependency? I'm not totally sure. Either way, thanks for removing it!